README.md

```diff
@@ -261,15 +261,8 @@ package managers:
 - `gem.txt`: run-time dependencies installed by gem.
 - `npm.txt`: run-time dependencies installed by npm.
 - `pip.txt`: a normal [pip `requirements.txt`][] file, for run-time
-  dependencies too.
-
-All of these files will install separately each line. See this example
```

Comment:
That's already supported by the standard requirements.txt file as you can see in the test afaik